src/components/Questions/GeneralQuestionManager/GeneralQuestionManager.tsx

```diff
@@ -205,7 +205,9 @@ export default function GeneralQuestionManager(props: QuestionManagerProps) {
         <Appbar.BackAction size={18} style={{ margin: 0 }} onPress={goBack} />
         <TextSubtitle>Go Back</TextSubtitle>
       </ButtonHeader>
-      {currentQuestions.map(question => getQuestionComponent(question))}
+      {currentQuestions.map((question, id) =>
+        getQuestionComponent(question, id),
+      )}
       {filledCase ? <p>Already filled out a form for this case.</p> : null}
```

Comment:
This is an issue from when I prematurely merged in a PR, but can be fixed in this branch:
- replace <p> with <TextRegular>
- import TextRegular from fonts/Fonts

Reply:
> Bug:
> 
> * it's possible to make duplicates for a case in different languages
>   
>   * Say you have Citizenship filled out for English. Then you switch your language pref to Espanol. The Intake Form flow allows you to fill another form for Citizenship, but it shouldn't.

Have a feeling this has to do with the `getVisitReason` function in GeneralQuestionsManager - I'm not 100% what this function's doing, but it seems like its checking only for EN visit types? 

```
 return visitReasons
      ? (visitReasons as MultilingualQuestion).EN[index]
      : null;
```